chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/edition2015/RequiredAndRelatedCriteriaReviewer.java

```diff
@@ -99,9 +99,9 @@ private void checkACriteriaHaveRequiredDependencies(CertifiedProductSearchDetail
         errors = validationUtils.checkClassSubsetOfCriteriaForMissingComplementaryCriteriaErrors(A_CRITERIA_NUMBERS_START,
                 attestedCriteria, exceptionsToRequiredByACriteria, exceptionsToACriteria);
         listing.getErrorMessages().addAll(errors);
-        warnings = validationUtils.checkClassSubsetOfCriteriaForMissingComplementaryCriteriaWarnings(A_CRITERIA_NUMBERS_START,
-                attestedCriteria, exceptionsToRequiredByACriteria, exceptionsToACriteria);
-        addListingWarningsByPermission(listing, warnings);
+        //warnings = validationUtils.checkClassSubsetOfCriteriaForMissingComplementaryCriteriaWarnings(A_CRITERIA_NUMBERS_START,
```

Comment:
I think you missed a section of stuff to remove. Github won't let me comment where there is no change, but check a few lines above... around line 95. This Reviewer is used in flexible upload too and I noticed warnings still in there when testing a different feature so came here to check the change.